worlds/wargroove2/Regions.py

```diff
@@ -0,0 +1,91 @@
+from BaseClasses import Region, Entrance
+from worlds.wargroove2.Levels import region_names, FINAL_LEVEL_1, FINAL_LEVEL_2, FINAL_LEVEL_3, FINAL_LEVEL_4
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from . import Wargroove2World
+
+
+def create_regions(world: "Wargroove2World") -> None:
+    multiworld = world.multiworld
+    player = world.player
+    level_list = world.level_list
+    first_level = world.first_level
+    final_levels = world.final_levels
+    
+    menu_region = Region('Menu', player, multiworld)
+    menu_region.exits.append(Entrance(player, 'Menu exits to Humble Beginnings Rebirth', menu_region))
+    first_level_region = first_level.define_region("Humble Beginnings Rebirth", multiworld,
+                                                   exits=[region_names[0], region_names[1],
+                                                          region_names[2], region_names[3]])
+    multiworld.regions += [menu_region, first_level_region]
+
+    # Define Level 1s
+    for level_num in range(0, 4):
+        next_level = level_num * 4 + 4 - level_num
+        multiworld.regions += [level_list[level_num].define_region(region_names[level_num], multiworld, exits=[
+            region_names[next_level],
+            region_names[
+                next_level + 1],
+            region_names[
+                next_level + 2]])]
```

Comment:
This is really just a readability comment, but I think this is a lot clearer/cleaner
```suggestion
        multiworld.regions += [level_list[level_num].define_region(region_names[level_num], multiworld, exits=[
            region_names[next_level],
            region_names[next_level + 1],
            region_names[next_level + 2]
        ])]
```